ui/src/main/webapp/src/app/executions/executions-layout.component.ts

```diff
@@ -91,6 +91,62 @@ export class ExecutionsLayoutComponent extends ProjectLayoutComponent implements
     }
 
     protected createContextMenuItems() {
+        this.submenuTechnologyItems = [
```

Comment:
What has happened here? I guess placeholders, but it will collide more than not having htem :)